src/core/store/index.ts

```diff
@@ -218,6 +218,24 @@ namespace Store {
     products: Product[];
   }
 
+  export type AvailableNavigations = Indexed<Navigation> & {
+    sortOrder: Recommendations.Navigation[];
+  };
+
+  export namespace Recommendations {
+    export interface Navigation {
+      name: string;
+      values: RecommendationRefinement[];
+      lows?: any[];
```

Comment:
why don't we have types for these?
we can still use range `lows` and `highs` to determine navigation sort order

Reply:
Sorry elaborate on what you mean by us not having types for these?